packaging/app.organicmaps.desktop.metainfo.xml

```diff
@@ -113,6 +113,32 @@
   </screenshots>
 
   <releases>
+   <release version="2023.08.18-8" date="2023-08-18">
+     <description>
+       <p>Highlights:</p>
+         <ul>
+           <li>New OpenStreetMap data as of August 14, 2023</li>
+           <li></li>
+           <li></li>
+         </ul>
+       <p>Search:</p>
+         <ul>
+           <li></li>
```

Comment:
Do you plan to add more details here? Please consider implementing a script which will read release-notes.txt from a file and generate this XML.

Reply:
> Do you plan to add more details here?

Yes, I was waiting for input, now I got it from [here](https://github.com/flathub/app.organicmaps.desktop/pull/55#discussion_r1299153801). 

> Please consider implementing a script which will read release-notes.txt from a file and generate this XML.

I am all for it, and I would gladly implment scripts, but that needs input from project leaders, which has to be provided on time (before release) otherwise the retroactive patching will still be needed for flatpak packages. [The discussion about the topic stalled here](https://github.com/orgs/organicmaps/discussions/4699#discussioncomment-5332703).